common/config/rush/pnpm-lock.yaml

```diff
@@ -7,7 +8,7 @@ dependencies:
   '@azure/ms-rest-nodeauth': 0.9.3
   '@microsoft/api-extractor': 7.2.1
   '@rush-temp/abort-controller': 'file:projects/abort-controller.tgz'
-  '@rush-temp/amqp-common': 'file:projects/amqp-common.tgz'
+  '@rush-temp/core-amqp': 'file:projects/core-amqp.tgz'
```

Comment:
Just to confirm, the changes in this file were auto-generated right? And we didnt manually change anything?

Reply:
Yes, these changes in the file were auto-generated.